src/categorical.py

```diff
@@ -45,6 +45,7 @@ def _label_binarization(self):
     def _one_hot(self):
         ohe = preprocessing.OneHotEncoder()
         ohe.fit(self.df[self.cat_feats].values)
+        self.ohe = ohe
```

Comment:
can you just do 

```
self.ohe = preprocessing.OneHotEncoder()
self.ohe.fit(self.df[self.cat_feats].values)
```
Thanks for the fix

Reply:
Thanks for the feedback Abhishek. Done :)